app/api/ringle_music/v1/user_api.rb

```diff
@@ -61,6 +61,137 @@ class UserApi < Grape::API
+                    end
+                    post do
+                        authenticate!
+                        begin
+                            UserMusicService::UserMusicsAdder.call(current_user, params[:music_ids])
+                        rescue ActiveRecord::RecordNotFound => e
+                            error!({ message: "User Not Found" })
+                        rescue => e
+                            error!({ message: e.message })
+                        end
+                    end
+
+                    desc 'delete music to playlist for user'
+                    params do
+                        requires :music_ids, type: Array[Integer], desc: "Array of music ids to add to the playlist"
+                    end
+
+                    delete do
+                        authenticate!
+                        begin
+                            UserMusicService::UserMusicsDestroyer.call(current_user, params[:music_ids])
+                        rescue ActiveRecord::RecordNotFound => e
+                            error!({ message: "User Not Found" })
+                        rescue => e
+                            error!({ message: e.message })
+                        end
+
+                    end
+
+                    desc 'get playlist of user'
+                    get do
+                        authenticate!
+
+
+                        musics = UserMusicService::UserMusicsGetter.call(current_user)
+                        present musics, with: Entities::MusicEntity
+
+
+                    end
+                end
+
+                resource :likes do
+                    resource :musics do
+                        get do
+                            authenticate!
+
+                            musics = UserLikesMusicService::UserLikesMusicsGetter.call(current_user)
+                            present musics, with: Entities::MusicEntity
+                        end
+
+
+                        route_param :music_id, type: Integer do
+                            delete do
+                                authenticate!
+            
+                                begin
+                                    UserLikesMusicService::UserLikesMusicCanceler.call(current_user, params[:music_id])
+            
+                                rescue ActiveRecord::RecordNotFound => e
+                                    return {success: false, message: "User doesn't like Found"}
+                                rescue => e
+                                    return {success: false, message: e.message}
+                                end
+                                
+            
+                            end
+        
+                            post do
+                                authenticate!
+                                begin
+                                    UserLikesMusicService::UserLikesMusicPoster.call(current_user, params[:music_id])
```

Comment:
보통 module name이 아닌 class name을 ~Service로 끝냅니다!
(저희 컨벤션도 그렇고, gitlab 레포 등 참고해봐도 그렇습니다)

```suggestion
UserLikesMusic::CreateService.call(current_user, params.symbolize_keys
```